docs/examples.md

```diff
@@ -78,7 +78,7 @@ This example will deploy a MinIO tenant with TLS using certificates provided by
   certificate keypairs:
 
   ```sh
-    mkcert "*.minio-tenant.svc.cluster.local"
+    mkcert "minio.minio-tenant.svc.cluster.local"
     mkcert "*.myminio.minio-tenant.svc.cluster.local"
```

Comment:
I think it should also work without this certificate. I don't think we will have any hostnames that will match it.